cdc/api/v1/api_test.go

```diff
@@ -785,9 +789,11 @@ func TestServerStatus(t *testing.T) {
 	require.Nil(t, err)
 	require.Equal(t, "capture-for-test", resp.ID)
 	require.True(t, resp.IsOwner)
+	require.Equal(t, "abcd", resp.ClusterID)
```

Comment:
Do we need to add `liveness` test cases?

Reply:
liveness test cases is already added